decidim-core/app/commands/decidim/invite_user_to_private_participatory_space.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A command with the business logic to invite a user to
+  # a private participatory space.
+  class InviteUserToPrivateParticipatorySpace < Rectify::Command
+    # Public: Initializes the command.
+    #
+    # form - A form object with the params.
```

Comment:
This comment doesn't make sense, isn't?

Reply:
Woops that's right, I used InviteUserAgain as a template to create this one. It means the comment there doesn't make sense either. I corrected both now